File: Examples/VisualBasic/NoteBook/CreatingPasswordProtectedDoc.vb
Imports System.IO
Imports Aspose.Note
Imports Aspose.Note.Saving
Imports System.Drawing
Imports System.Globalization

Namespace WorkingWithNoteBook
    Public Class CreatingPasswordProtectedDoc
        Public Shared Sub Run()
            ' ExStart:CreatingPasswordProtectedDoc
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_NoteBook()

            Dim document As New Document()
            document.Save(dataDir & Convert.ToString("CreatingPasswordProtectedDoc_out.one"), New OneSaveOptions() With {
                 .DocumentPassword = "pass"
            })
            ' ExEnd:CreatingPasswordProtectedDoc       
        End Sub
    End Class
End Namespace

File: Examples/CSharp/NoteBook/CreatingPasswordProtectedDoc.cs
using System.IO;
using Aspose.Note;
using Aspose.Note.Saving;
using System;
using System.Drawing;
using System.Globalization;

namespace Aspose.Note.Examples.CSharp.WorkingWithNoteBook
{
    public class CreatingPasswordProtectedDoc
    {
        public static void Run()
        {
            // ExStart:CreatingPasswordProtectedDoc
            // ExFor:Document
            // ExFor:OneSaveOptions
            // ExFor:OneSaveOptions.DocumentPassword
            // ExSummary:Shows how to save document with encryption.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_NoteBook();

            Document document = new Document();
            document.Save(dataDir + "CreatingPasswordProtectedDoc_out.one", new OneSaveOptions() { DocumentPassword = "pass" });

            // ExEnd:CreatingPasswordProtectedDoc       
        }
    }
}
